packages/cocoon_server_test/test/google_auth_provider_test.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:cocoon_server_test/google_auth_provider.dart';
+import 'package:http/http.dart' as http;
+import 'package:http/testing.dart' as http_testing;
+import 'package:test/test.dart';
+
+void main() {
+  test('delegates withCreateClient (no baseClient)', () async {
+    late final List<String> capturedScopes;
```

Comment:
just `final capturedScopes = <String>[];` and then `capturedScopes.addAll(scopes)` below would capture if this was being called twice (e.g. ['foo', 'foo'])

Reply:
Ok dokie. Done.